edx_notifications/server/web/static/edx_notifications/js/views/counter_icon_view.js

```diff
@@ -132,9 +132,8 @@ var CounterIconView = Backbone.View.extend({
                  notification_alert.play();
              }
              counterView.render();
-             if (counterView.notification_pane) {
-                 counterView.notification_pane.collection.url = counterView.options.endpoints.user_notifications_unread_only;
-                 counterView.notification_pane.hydrate();
+             if (counterView.notification_pane && counterView.notification_pane.selected_pane == "unread") {
```

Comment:
I'm not sure I'm following this logic. Can you explain? Also, in general I'd prefer that you directly call into javascript rather than simulating a mouse click. Thanks.